roles/bind-keycloak-apb/tasks/main.yml

```diff
@@ -24,23 +22,63 @@
   retries: 600
   delay: 2
 
--
-  name: "Create {{ service }}-bearer client in realm {{ namespace }}"
+- name: Get Keycloak Realms
```

Comment:
Here you can check if [realm exists](https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/provision-keycloak-apb/tasks/configure-keycloak.yml#L18-L40). But that only other way how todo that. This one is OK to me